xml/admin_monitoring_alerting.xml

```diff
@@ -0,0 +1,1065 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"  xml:id="monitoring-alerting">
+ <title>Monitoring and Alerting</title>
+ <para>
+  In &productname; 6, &deepsea; no longer deploys a monitoring and alerting
+  stack on the &smaster;. Users have to define the &prometheus; role
+  for &prometheus; and &alertmanager;, and the &grafana; role for &grafana;.
+  When multiple nodes are assigned with the &prometheus; or &grafana; role, a
+  highly available setup is deployed.
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    <emphasis role="bold">&prometheus;</emphasis> is the monitoring and alerting
+    toolkit.
+   </para>
+ </listitem>
+ <listitem>
+  <para>
+   <emphasis role="bold">&alertmanager;</emphasis> handles alerts sent by
+   the &prometheus; server.
+  </para>
+ </listitem>
+  <listitem>
+   <para>
+    <emphasis role="bold">&grafana;</emphasis> is the visualization and alerting
+    software.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    The <systemitem class="daemon">prometheus-node_exporter</systemitem>
+    is the service running on all &sminion;s.
+   </para>
+  </listitem>
+ </itemizedlist>
+ <para>
+  The &prometheus; configuration and <emphasis>scrape</emphasis> targets
+  (exporting daemons) are setup automatically by &deepsea;. &deepsea; also
+  deploys a list of default alerts, for example <literal>health
+  error</literal>, <literal>10% OSDs down</literal>, or <literal>pgs
+  inactive</literal>.
+ </para>
+
+ <sect1 xml:id="pillar-variables">
+  <title>Pillar Variables</title>
+  <para>&spillar; contains information about a minion or many minions stored or
+    generated on the &smaster;. The pillar is already running in &salt; by
+    default. There are two main files:</para>
```

Comment:
Suggestion:
The pillar is a key-value store that provides information and configuration values to minions. It is availabel to all minions though potentially with different content. The pillar is pre-populated with default values and can be customized in two different ways: